### DISCUSSION TITLE
Keeping “stacks” up to date post-installation

### DISCUSSION THREAD

**Discussion by moishinetzer** [epicweb-dev/epic-stack#79] [General]
I'm not sure if this discussion is better situated at the blues-stack or the indie-stack but given this stack has the most engagement I thought I would post it here. 

After relying on the indie-stack for a long time a big pain point for me was keeping my stack updated. This meant when new features or bug fixes were rolled out I would have to manually figure out what was changed and apply that to my project. 

I dont know what the solution to this would be however I'm sure this is an issue that other remix-stack developers resonate with. I hope the epic-stack can figure out a way to address this given the roadmap seems to have lots of promising new technologies that will probably require reconfiguring the project to suit it.

**Comment by kentcdodds** [comment]
The only solution here is to take bits and pieces and make them libraries you install instead of code that's generated. Unfortunately, a lot of what's generated isn't very useful as generic utilities.

I understand this is a problem, but I don't believe solving it is practical.

**Comment by kentcdodds** [comment]
I added documentation for this: https://github.com/epicweb-dev/epic-stack/blob/main/docs/managing-updates.md
